include/beman/any_view/any_view.hpp

```diff
@@ -3,7 +3,7 @@
 #ifndef BEMAN_ANY_VIEW_ANY_VIEW_HPP
 #define BEMAN_ANY_VIEW_ANY_VIEW_HPP
 
```

Comment:
Why is this needed? 

We have `beman.optional` integrate on Compilter Explorer and it works: `#include <beman/optional/optional.hpp>`
* https://github.com/bemanproject/optional/tree/main/examples
* https://godbolt.org/z/ebEzYc5fv


Please also check 
* https://github.com/compiler-explorer/infra/blob/main/bin/yaml/libraries.yaml#L1156-L1163
* https://github.com/compiler-explorer/compiler-explorer/blob/main/etc/config/c%2B%2B.amazon.properties#L4094-L4098

Let me know how can I help. You can show me draft PRs in boths repos for compiler-explorer.

Reply:
L.E. @RaduNichita offered do to that in https://github.com/bemanproject/beman/issues/110.

Reply:
Thanks, I'll go ahead and close this PR then, looks like the ball's rolling on this deployment approach.

Reply:
Ok - and thanks for contribution anyways!

@patrickroberts , since I finished the tutorial, Patrick, is deployment of any_view something you are interested in?

@RaduNichita can pick other left library.